### DISCUSSION TITLE
win32-64-mgen-5.1.1.zip

### DISCUSSION THREAD

**Discussion by ljt-git** [USNavalResearchLaboratory/mgen#32] [General]
Mgen 5.1.1 windows release.  Includes win32 and win64 binaries.

<hr /><em>This discussion was created from the release <a href='https://github.com/USNavalResearchLaboratory/mgen/releases/tag/'>win32-64-mgen-5.1.1.zip</a>.</em>

**Comment by ducleotide** [comment]
It seems like this is not a zip file but a tarball.

**Reply by weston-nrl** [reply]
I just fixed it manually.  Untarred, zipped, and re-uploaded
